2.train-VAE/2.fetch_hyper_params.ipynb

```diff
@@ -67,13 +67,13 @@
      "output_type": "stream",
```

Comment:
<p>Consider writing these to a <code>json</code> or <code>yaml</code> file for access in the training notebook to avoid hardcoding of hyperparameters.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
I'm going to hold off on this for now since we are not only interested in the #1 optimal hyperparameter, but also hyperparameter sets with similar val_loss but greater `latent_dim` and `beta` values